src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$card-hover-text-color: #34568b;
+$nav-hover-text-color: #00acdc;
```

Comment:
do you really need to give them variables name? 
because the colors only used once

Reply:
For this task, yes they are used once. But if it is a bigger project we will need to give them variables. Therefore I decided to give them variables. It isn't necessary for this specific task though.